gp-populate-anything/gppa-ignore-current-user-filter-for-admins.php

```diff
@@ -25,7 +25,7 @@
 		return $query_builder_args;
 	}
 
-	array_pop( $query_builder_args['where'][ $filter_group_index ] );
```

Comment:
Based on this usage, this is wiping out the entire `where` and effectively removing all filters from what I can see.

My guess is that the issue has to do with the filter using `special_value:current_user:ID` not being the last filter in the group.

You'll need to loop through `$query_builder_args['where'][ $filter_group_index ]` and find the filter that has `$filter['value'] === 'special_value:current_user:ID'` and remove it from the `$query_builder_args['where'][ $filter_group_index ]` array.

Reply:
Okay. Actually $filter_group_index does that job fine. What is happening specifically in this test scenario is that there is only 1 filter and that is using `special_value:current_user:ID`. So unsetting messes up the overall structure of the $query_builder_args which is what I've fixed in the next test and pushed it. @claygriffiths